### PR TITLE
commit

### PR DESCRIPTION
**Pull request by nareshsingh8**

### CODE REVIEW

File: app/views/albums/_form.html.erb [hunk 1]
```diff
@@ -21,7 +21,7 @@
        <% end %>
     </div>
     <br>
-    <strong><label>Photos:</label></strong><br>
+    <strong><label>Album Photos:   </label></strong><br>
```
Comment (gshika) [inline app/views/albums/_form.html.erb:24]: Naresh what is this, why are u giving spaces unnecessary , don't you love money wanrt to  loose jobl,lm,;,,;,;

File: app/views/albums/index.html.erb [hunk 2]
```diff
@@ -33,6 +33,8 @@
 </div>
 
 
+<%# test comments %>
```
Comment (gshika) [inline app/views/albums/index.html.erb:36]: test2

File: app/views/albums/_form.html.erb (hunk 1 above)
Comment (nareshsingh8) [reply app/views/albums/_form.html.erb:24]: sorry to say but i'm literally frustrated by your 5000 job. GIve me some space as i did.

File: app/views/albums/_form.html.erb [hunk 3]
```diff
@@ -21,7 +21,7 @@
        <% end %>
     </div>
     <br>
-    <strong><label>Photos:</label></strong><br>
```
Comment (gshika) [inline app/views/albums/_form.html.erb:24]: ???????

**Pull request merged**